roles/allocations/templates/allocations_balance_service.conf.j2

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+debug: False
```

Comment:
Is this syntax right? For some reason I thought it needed `=` instead of `:`. Can it do both? Am I crazy?? :D

Reply:
I think you asked this before:) `:` works.

Reply:
I am going crazy then, good to know! 🤪